### PR TITLE
feat(account-lib): add encodeAddress in account-lib and unit test

### PR DESCRIPTION
**Pull request by JoaquinTissera-Simtlix**
add encodeAddress in account-lib and unit test

STLX-4525

### CODE REVIEW

File: modules/account-lib/src/coin/algo/utils.ts [hunk 1]
```diff
@@ -292,6 +292,16 @@ export class Utils implements BaseUtils {
     return algosdk.encodeAddress(addr);
   }
 
+  /**
+   * decodeAddress return an addres decoding with algosdk
+   *
+   * @param addr
+   * @returns Uint8Array
+   */
+  decodeAddress(addr: string): any {
```
Comment (rliberal-simtlix) [inline modules/account-lib/src/coin/algo/utils.ts:301]: try to return here Uinit8array not any

File: modules/account-lib/test/unit/coin/algo/utils.ts [hunk 2]
```diff
@@ -165,4 +165,10 @@ describe('utils', () => {
     const feeRate = -1;
     should.throws(() => utils.getFeeData(payTxn, feeRate), 'Error: FeeRate must be integer positive number ');
   });
+
+  it('should decode address', () => {
```
Comment (rliberal-simtlix) [inline modules/account-lib/test/unit/coin/algo/utils.ts:169]: Add failure case

**Review by rliberal-simtlix — CHANGES_REQUESTED** [review]

File: modules/account-lib/src/coin/algo/utils.ts (hunk 1 above)
Comment (JoaquinTissera-Simtlix) [reply modules/account-lib/src/coin/algo/utils.ts:301]: It happens that it does not return a uint8array but an addres object that has a publicKey and the checksum

**Review by rliberal-simtlix — APPROVED** [review]

File: modules/account-lib/src/coin/algo/utils.ts [hunk 3]
```diff
@@ -292,6 +292,16 @@ export class Utils implements BaseUtils {
     return algosdk.encodeAddress(addr);
   }
 
+  /**
+   * decodeAddress return an addres decoding with algosdk
+   *
+   * @param addr
+   * @returns Uint8Array
```
Comment (rliberal-simtlix) [inline modules/account-lib/src/coin/algo/utils.ts:299]: change return here

**Review by rliberal-simtlix — CHANGES_REQUESTED** [review]

File: modules/account-lib/test/unit/coin/algo/utils.ts (hunk 2 above)
Comment (JulioAchaval-SimTLiX) [reply modules/account-lib/test/unit/coin/algo/utils.ts:169]: Same

**Review by nicolasbellanich-simtlix — APPROVED** [review]

**Review by rliberal-simtlix — APPROVED** [review]

**Review by JulioAchaval-SimTLiX — APPROVED** [review]
LGTM

**Review by pancham88 — APPROVED** [review]

**Pull request merged**